Makefile

```diff
@@ -54,16 +48,23 @@ format-check: ## Checks if format is correct
 
 .PHONY: watch
 watch: ## Watch for the filesystem and rebuild on every change
-	$(DUNE) build --watch @react @node
+	$(DUNE) build --watch
 
 .PHONY: test
 test: ## Run test suite
-	$(DUNE) test --no-buffer
+	$(DUNE) build @runtest --no-buffer
 
 .PHONY: docs
 docs: ## Build the docs
-	$(DUNE) build @doc
+	rm -rf _docs
```

Comment:
I wish this could be a single command but I don't think it's possible

Reply:
There might be a way to do this in a single command by leveraging Dune context, profiles and env vars.

In a new `dune-workspace`, add this:

```
(context (default (profile ml)))
(context (reason (profile re)))

; define env vars
(env
 (ml
  (env-vars
   (ODOC_SYNTAX ocaml))))
(env
 (re
  (env-vars
   (ODOC_SYNTAX reason))))
```

Then, a single run of `dune build @doc` should put the OCaml docs in `_build/default/_doc/_html` and the Reason ones in `_build/reason/_doc/_html/`.

Reply:
I tried to get this to work, but ran into too many errors. Here's the last version I got before giving up:

```
(lang dune 3.8)

(context
 (default
  (profile ml)))

(context
 (reason
  (profile re)))

; define env vars

(env
 (ml
  (env-vars
   (ODOC_SYNTAX ocaml)))
 (re
  (env-vars
   (ODOC_SYNTAX reason))))
```
Running `opam exec -- dune build @doc` gave this error:

```
File "dune-workspace", line 8, characters 2-8:
8 |  (reason
      ^^^^^^
Error: Unknown constructor reason
make: *** [Makefile:59: docs] Error 1
```